11LessonEleven/in-class-project/index.html

```diff
@@ -1,10 +1,63 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Intel</title>
+  <title>Lesson 11 | In Class</title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
 
+<!--   Welcome Section   -->
+
+  <section id="welcome">
+    <h1>Welcome to our site</h1>
+  </section>
+
+<!--  Title Boxes Section  -->
+
+  <section id="titles" class="clearfix">
+
+<!--    Left Box   -->
+
+    <div class="title">
+      <div class="titleColorL">
+        <!--Color Box -->
+      </div>
+
+      <div class="titleTxt">
+        <h2>Title</h2>
+        <p>Bacon ipsum dolor amet pig porchetta tongue tenderloin ham hock cupim strip steak sausage beef hamburger meatball ribeye. Landjaeger drumstick</p>
+      </div>
+    </div>
+
+<!--    Middle Box   -->
+
+    <div class="title middleBox">
+      <div class="titleColorM">
+        <!--Color Box -->
+      </div>
+
+      <div class="titleTxt">
+        <h2>Title</h2>
+        <p>Bacon ipsum dolor amet pig porchetta tongue tenderloin ham hock cupim strip steak sausage beef hamburger meatball ribeye. Landjaeger drumstick</p>
+      </div>
+    </div>
+
+<!--   Right Box   -->
+
+    <div class="title">
+      <div class="titleColorR">
+        <!--Color Box -->
+      </div>
+
+      <div class="titleTxt">
+        <h2>Title</h2>
+        <p>Bacon ipsum dolor amet pig porchetta tongue tenderloin ham hock cupim strip steak sausage beef hamburger meatball ribeye. Landjaeger drumstick</p>
+      </div>
+    </div>
+
+  </section>
 </body>
 </html>
```

Comment:
Looks like there is some centering issues when you transition from different media queries. Please fix and I can update your grade.
<img width="634" alt="screen shot 2016-09-13 at 4 03 02 pm" src="https://cloud.githubusercontent.com/assets/12698279/18491369/ba4a5348-79cb-11e6-85d4-27d44361d9b3.png">
<img width="791" alt="screen shot 2016-09-13 at 4 02 56 pm" src="https://cloud.githubusercontent.com/assets/12698279/18491368/ba4a31f6-79cb-11e6-8d1e-cc3c71acd322.png">